validator/tests/test_static.py

```diff
@@ -256,6 +265,15 @@ def test_invalid_geo_id_hrr(self):
         assert len(report.raised_errors) == 1
         assert report.raised_errors[0].check_name == "check_bad_geo_id_value"
 
+    def test_invalid_geo_id_hhs(self):
```

Comment:
duplicate test name of line 155

Reply:
Changed naming convention to make the test names globally unique